src/graphiql-fetcher.ts

```diff
@@ -0,0 +1,32 @@
+import { SubscriptionClient } from './client';
+
+export const graphQLFetcher = (subscriptionsClient: SubscriptionClient, fallbackFetcher: Function) => {
+  let activeSubscriptionId: number | null = null;
+
+  return (graphQLParams: any) => {
+    if (subscriptionsClient && activeSubscriptionId !== null) {
+      subscriptionsClient.unsubscribe(activeSubscriptionId);
+    }
+
+    if (subscriptionsClient && graphQLParams.query.startsWith('subscription')) {
```

Comment:
As this is for GraphiQL I don't think the added weight of pulling in GraphQL is too much of a concern.